packages/component-library/src/BaseMap/BaseMap.js

```diff
@@ -182,18 +188,32 @@ class BaseMap extends Component {
         ? CIVIC_DARK
         : CIVIC_LIGHT;
 
+    const animationProps = !animate
+      ? {}
+      : {
+          transitionDuration: 1000,
+          transitionInterpolator: new FlyToInterpolator()
+        };
+
     return (
       <div className={mapWrapper}>
         <MapGL
           className="MapGL"
           {...viewport}
+          {...animationProps}
           mapStyle={baseMapboxStyleURL}
           mapboxApiAccessToken={mapboxToken}
           onViewportChange={newViewport => this.onViewportChange(newViewport)}
           ref={this.mapRef}
           {...mapGLOptions}
           onClick={onBaseMapClick}
           onLoad={onMapLoad}
+          scrollZoom={isInteractive}
+          dragPan={isInteractive}
+          dragRotate={isInteractive}
+          doubleClickZoom={isInteractive}
+          touchZoom={isInteractive}
+          touchRotate={isInteractive}
```

Comment:
@mendozaline, I can see in this [example](https://hackoregon.github.io/civic/?path=/story/component-lib-maps-base-map--example-no-interactivity) where `mapGLOptions` is used to prevent interaction. If this can be set 2 ways with props I think that will be confusing for future developers. That being said. I've been snooping around and I can't seem to find any documentation that defines what can be in `mapGLOptions` so I'm not entirely clear on what that prop can do.

Reply:
If you click on the Notes tab next to the Canvas tab in the upper-left corner of the Base Map story, you'll see what options can be set with the `mapGLOptions` prop under the "Example: No Interactivity" section. These are just some of the the most common options we've used before, but it also includes a link to the `react-map-gl` [documentation](https://github.com/uber/react-map-gl/blob/master/docs/components/interactive-map.md) that has even more options that can be set with it.

![Screenshot_2019-08-12 Storybook](https://user-images.githubusercontent.com/9361258/62897562-64cc2800-bd08-11e9-8b70-e984cc9a0ede.png)